### PR TITLE
Debounce deletion of un-referenced attachments --> don't fixes but mi…

### PR DESCRIPTION
**Pull request by ehhc**
Debounce deletion of un-referenced attachments --> don't fixes but mitigates the problems of #3103

### CODE REVIEW

**Comment by ZeroX-DG** [comment]
@Rokt33r What do you think about this solution? Is there a better way to deal with this?

File: browser/main/lib/dataApi/attachmentManagement.js [hunk 1]
```diff
@@ -584,6 +584,7 @@ function deleteAttachmentFolder (storageKey, noteKey) {
  * @param noteKey NoteKey of the current note. Is used to determine the belonging attachment folder.
  */
 function deleteAttachmentsNotPresentInNote (markdownContent, storageKey, noteKey) {
+  console.log('deleteAtt')
```
Comment (Rokt33r) [inline browser/main/lib/dataApi/attachmentManagement.js]: This one!

Comment (ehhc) [reply browser/main/lib/dataApi/attachmentManagement.js]: ups.. stupid me... i thought i had removed that..

Comment (ehhc) [reply browser/main/lib/dataApi/attachmentManagement.js]: done

**Pull request merged**